tools/shared.py

```diff
@@ -2538,6 +2539,11 @@ def closure_compiler(filename, pretty=True, advanced=True, extra_closure_args=[]
       java_home = os.path.dirname(JAVA)
       if java_home:
         env.setdefault('JAVA_HOME', java_home)
+      if WINDOWS and '--platform' not in user_args:
```

Comment:
Why this `user_args` scheme? If I build from command line with `em++ a.c -o a.html -O3 --closure 1 --closure-args "--platform native"`, it won't work?

Reply:
Oh yes I forgot about that.  We should support that.  I'm just hoping that passing all those other args along with `--version` will be ok.

In the mean time it looks like "--version --platform=java" still tries to run the native version for some reason :(    So out windows build is still broken.    Seems like the google-closure npm pacakge has a lot of issues outstanding.